fs/read.js

```diff
@@ -1,16 +1,32 @@
-var fs = require('fs');
+var fs = require("fs");
 
 // Use fs module for all operatins below.
 
-
 // 1. Write script to read file theory.md and console the output buffer.
+fs.readFile("./theory.md", (err, data) => {
+  if (err) return err;
+  return console.log(data);
+});
 
 // 2. Write script to read file theory.md and convert resulted buffer using toString
-  // method and console the result.
+// method and console the result.
+fs.readFile("./theory.md", (err, data) => {
+  if (err) return err;
+  return console.log(data.toString());
+});
 
 // 3. Write script to read file Synchronously and console the output.
+var data = fs.readFileSync("./theory.md");
+console.log(data);
 
 // 4. Write script to create a file 'write.js' and write content of read.js in there.
+fs.open("/write.js", "wx", (err, fd) => {
+  if (err) return err;
+  fs.write(fd, data, err => {
```

Comment:
Here data should be the content of read.js file.
```js
fs.readFile('./read.js', (err, data) => {
  fs.writeFile(fd, data, err =>  {})
})
```
You are using fs.write which is different from writeFile method.
Check the difference between write and writeFile method.